features/promotion/reapplying_promotion_on_cart_change.feature

```diff
@@ -5,23 +5,23 @@ Feature: Reapplying promotion on cart change
     I want to have proper discount applied after every operation on my cart
 
     Background:
-        Given the store is operating on a single "France" channel
+        Given the store operates on a single channel in "France"
         And the store has a product "PHP T-Shirt" priced at "€100.00"
         And there is a promotion "Holiday promotion"
         And I am logged in customer
 
-    @todo
+    @ui
     Scenario: Not receiving discount on shipping after removing last item from cart
         Given the store has "DHL" shipping method with "€10.00" fee
         And the promotion gives "100%" percentage discount on shipping to every order
         And I have product "PHP T-Shirt" in the cart
         When I proceed selecting "DHL" shipping method
         And I remove product "PHP T-Shirt" from the cart
-        Then my cart total should be "€0.00"
-        And my cart shipping fee should be "€0.00"
-        And my discount should be "€0.00"
+        Then cart total should not be displayed
```

Comment:
`displayed` is coupled to `UI` we should find better step definition. Let's brainstorm on that.
`Then cart should be empty with no value` - do anyone have a better idea?